tests/acceptance/steps/watermark_get.py

```diff
@@ -16,8 +16,7 @@
 
 from behave import then
 from behave import when
-
-from .util import call_watermark_get
+from util import call_watermark_get
```

Comment:
That's weird. I'll merge your branch and take a look at that. It's mostly a nitpick.